src/components/__tests__/box_placeholder.test.tsx

```diff
@@ -16,7 +16,13 @@ describe('The BoxPlaceholder component', () => {
         wrapper = shallow(<BoxPlaceholder {...props} />);
     });
 
-    it('should render', () => {
+    it('should render with className "background--transparent"', () => {
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Maybe a whole snapshot is a bit overkill here and we just wanna test the css class property for it's value?

Reply:
Thanks for the suggestion: now it only checks whether the expected class has been included or not..
https://github.com/Ultimaker/react-web-components/pull/246/files#diff-83f623b360f5c357d9712ace54ffc68fR26